pkg/client/testdata/buildpack-flatten/buildpack-7/buildpack.toml

```diff
@@ -0,0 +1,9 @@
+api = "0.3"
```

Comment:
are you using buildpack api 0.3 on purpose in these?

Reply:
Not really. I think I copied and paste from another similar file